schemas/bedrock-vc-issuer.js

```diff
@@ -77,6 +77,9 @@ export const issueCredentialBody = {
           items: {
             type: 'string'
           }
+        },
+        extraInformation: {
+          type: 'string'
```

Comment:
So this is going to be a string -- but it should be base64url-encoded and it's got to be decoded once received. We should be performing that decoding somewhere, perhaps in the above factory function for the cryptosuite. We could add a regex for that sort of thing here, but just running the decoder would probably be simpler since it has to run anyway (we don't need double validation).

```suggestion
          type: 'string'
```

Reply:
To further clarify, `extraInformation` holds bytes, but we have to transfer it as JSON, so it needs to be encoded as a string -- and the preferred encoding for that is base64url. We usually use this lib (https://www.npmjs.com/package/base64url-universal) to provide the encoder (in the test client) and the decoder (here).